.github/workflows/nightly-sigar.yml

```diff
@@ -0,0 +1,25 @@
+name: Akka Nightly Sigar Build
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  sbt:
+    name: Test Akka Cluster Metrics
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8-0
+      - name: Test
+        run: |-
+          cp .sbtopts-nightly-sigar .sbtopts
+          sbt clean 
+          sbt akka-cluster-metrics/test
```

Comment:
Aren't these ran as part of the 'regular' nightly tests? What is the motivation for running them separately? Can we add that as a comment?

(I understand you might not know, but I don't know either, might be good to understand before we add this job?)

Reply:
This is with the sigar library enabled though.